app/src/main/java/com/pajato/android/gamechat/exp/Checkerboard.java

```diff
@@ -199,4 +209,8 @@ private int getDips(final DisplayMetrics metrics, final int px) {
         return (px * DisplayMetrics.DENSITY_DEFAULT) / metrics.densityDpi;
 
     }
+    private int convertSPtoDP(final int spValue, final DisplayMetrics displayMetrics) {
```

Comment:
It would be good to follow RNF and add a blank line and inline doc comment (/** ... */) here.